### DISCUSSION TITLE
[Accounts] set custom collection

### DISCUSSION THREAD

**Discussion by dr-dimitru** [meteor/meteor#12544] [Feature Requests]
- From: https://github.com/meteor/meteor-feature-requests/issues/20

Add custom option to Accounts package:

- `collection` {*String*|*Mongo.Collection*} - Instance of *Mongo.Collection* or collection name as *String*

**Comment by dr-dimitru** [comment]
Ready to send a PR if supported

**Reply by StorytellerCZ** [reply]
Yes, I would be interested to see it.

**Comment by dr-dimitru** [comment]
@StorytellerCZ thank you for the support, please review [this PR](https://github.com/meteor/meteor/pull/12591) by @dmromanov

**Comment by dr-dimitru** [comment]
Solved in https://github.com/meteor/meteor/pull/12591

@dmromanov @StorytellerCZ @Grubba27 thank you guys 🚀
